src/sections/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { Footer, SelectedListItem, ProfilePic } from "./components/index";
+import { styles } from "../../styles/styles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    justifyContent: "space-between",
+    flexDirection: "column",
+    width: styles.sizes.sidebarDimensions.width,
+    minWidth: styles.sizes.sidebarDimensions.minWidth,
+    maxWidth: styles.sizes.sidebarDimensions.maxWidth,
+    height: "100%",
+    backgroundColor: styles.colors.darkGray,
+    borderRight: `${styles.sizes.thickness.thick1}px solid black`
```

Comment:
can you make thicks have the `px `on it so that you don't have to add it everytime?